test/phantom-test.js

```diff
@@ -33,7 +33,6 @@ describe('phantom', function () {
       assert.equal(stdout.indexOf('# phantomjs:\n'
         + '1..1\n'
         + 'not ok 1 test fails\n'
-        + '  Error: Oh noes!\n'
```

Comment:
That's weird. It was broken for me on OS X with mocha@2.2.5 installed. What version do you have installed? Feel free to revert.